malariagen_data/anoph/plotly_params.py

```diff
@@ -215,3 +215,13 @@
     ],
     "Scope of the map.",
 ]
+
+taxon: TypeAlias = Annotated[
+    Optional[str],
+    "The taxon to restrict the dataset to.",
+]
+
+area: TypeAlias = Annotated[
+    Optional[str],
+    "The area to restrict the dataset to.",
+]
```

Comment:
Maybe these params should go in the `frq_params` module to be together with other parameters related to frequency calculation and visualisation?